db/builder.cc

```diff
@@ -336,6 +336,7 @@ Status BuildTable(
       // we will regrad this verification as user reads since the goal is
       // to cache it here for further user reads
       ReadOptions read_options;
+      read_options.rate_limiter_priority = Env::IO_USER;
```

Comment:
Why is the background operation file read-back user-level priority? I think the comment meant to describe why we allow block cache to be populated when reading back a new file (because it'll be accessed by user reads in the near future), not saying the readback needs to happen urgently.

Reply:
Will address this in a follow up PR.